service/static/index.html

```diff
@@ -13,6 +13,79 @@
       <div class="page-header">
           <h1>Order Demo REST API Service</h1>
       </div>
+
+      <!-- Flash Message -->
+      <div class="table-responsive">
+        <table class="table">
+          <tr><td>Status:</td><td><span id="flash_message"></span></td></tr>
+        </table>
+      </div>
+
+      <!-- FORM -->
+      <div class="col-md-12" id="form_data">
+        <h3>Create, Retrieve, Update, and Delete an Order:</h3>
+        <div class="well">
+          <div class="form-horizontal">
+            <div class="form-group">
+              <label class="control-label col-sm-2" for="order_id">Order ID:</label>
+              <div class="col-sm-6">
+                <input type="text" class="form-control" id="order_id" placeholder="Enter ID of Order">
+              </div>
+            </div>
+          </div> <!-- form horizontal -->
+
+          <div class="form-horizontal">
+
+            <!-- CUSTOMER ID -->
+            <div class="form-group">
+              <label class="control-label col-sm-2" for="order_customer_id">Customer ID:</label>
+              <div class="col-sm-10">
+                <input type="text" class="form-control" id="order_customer_id" placeholder="Enter customer ID for Order">
+              </div>
+            </div>
+
+            <!-- STATUS -->
+            <div class="form-group">
+              <label class="control-label col-sm-2" for="order_status">Status:</label>
+              <div class="col-sm-10">
+                <select class="form-control" id="order_status">
+                  <option value="CONFIRMED" selected>Confirmed</option>
+                  <option value="IN_PROGRESS">In Progress</option>
+                  <option value="SHIPPED">Shipped</option>
+                  <option value="DELIVERED">Delivered</option>
+                  <option value="CANCELLED">Cancelled</option>
+                </select>
+              </div>
+            </div>
+
+            <!-- CREATED ON -->
+            <div class="form-group">
+              <label class="control-label col-sm-2" for="order_created_on">Created On:</label>
+              <div class="col-sm-10">
+                <input type="text" class="form-control" id="order_created_on" readonly>
+              </div>
+            </div>
+
+            <!-- UPDATED ON -->
+            <div class="form-group">
+              <label class="control-label col-sm-2" for="order_updated_on">Gender:</label>
+              <div class="col-sm-10">
+                <input type="text" class="form-control" id="order_updated_on" readonly>
+              </div>
+            </div>
```

Comment:
Please remove these fields from the form, these are internal database fields and should not be displayed on UI.